public/main.js

```diff
@@ -53,6 +58,14 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        completeCounter.className = "complete-count";
+        if (count === 0) {
+            completeCounter.style.visibility = "hidden";
```

Comment:
Can't be styling in the JS. You'll have to add and remove a 'hidden' class on the object. I'd suggest looking at `String.split` and `Array.filter`